core/vm/instructions.go

```diff
@@ -726,7 +727,7 @@ func opCreate2(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
 
 	// Apply EIP150
 	gas -= gas / 64
-	scope.Contract.UseGas(gas, evm.Config.Tracer, tracing.GasChangeCallContractCreation2)
+	scope.Contract.UseMultiGas(multigas.ComputationGas(gas), evm.Config.Tracer, tracing.GasChangeCallContractCreation)
```

Comment:
Here the `tracing.GasChangeCallContractCreation2` reason has been changed to `tracing.GasChangeCallContractCreation`, was this intentional? 

I think it wasn't, I can open a PR to put back to it's rightful value, aligned with https://github.com/ethereum/go-ethereum/blob/master/core/vm/instructions.go#L712